### ISSUE TITLE
Remote - SSH Fails to connect

### ISSUE THREAD

**Issue by UrkoAT** [microsoft/vscode#192740]
Type: <b>Bug</b>

After It connects to a remote machine (using Remote - SSH) with no problem, I try to open a Folder, either from recent or manually opening in the remote start, loads the folder but then after in the bottom right corner starts to connect to the vscode server it fails, opening a modal dialog with "Cannot reconnect. Please reload the window" (tried on several machines, but same result). Im assuming its a client problem, because my co-workers can connect with no issue. I rebooted, uninstalled, deleted .config/Code, killed and removed the .vscode-server on the machines im trying to connect etc. I cannot figure what may be causing this. 

VS Code version: Code 1.82.0 (8b617bd08fd9e3fc94d14adb8d358b56e3f72314, 2023-09-06T22:07:18.759Z)
OS version: Linux x64 6.2.0-32-generic
Modes:
Connection to 'SSH: 10.91.3.30' could not be established  Canceled

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 4000)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|2, 1, 1|
|Memory (System)|15.40GB (12.28GB free)|
|Process Argv|--unity-launch --crash-reporter-id 10dfea37-e389-4367-a97b-2439d8eb1490|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|

Connection to 'SSH: 10.91.3.30' could not be established  Canceled
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
py29gd2263cf:30792227
vscaac:30438847
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30803844
3biah626:30602489
f6dab269:30613381
showlangstatbar:30737416
962ge761:30831758
03d35959:30757346
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxpt:30805730
pythonnoceb:30805159
dsvsc013:30795093
dsvsc014:30804076
diffeditorv1:30821571
dsvsc015:30829745

```

</details>

**Comment by UrkoAT** [comment]
I was able to resolve it for temporally changing the SSH Extension version to "pre release version". As mentioned by @jaywoojo in #192716 . Remote - SSH  is now working again.

**Comment by Phlya** [comment]
Pre-release version of the SSH extension also helped me with the same problem!

**Comment by ThreeCatsLoveFish** [comment]
Meet with the same problem and pre-release version of the SSH extension also helped me~

**Comment by irmo322** [comment]
I had the same problem, even with pre-release version.
I solved it by using the 0.105.1 version of the "Remote - SSH" extension.

**Comment by roblourens** [comment]
This should be resolved, if not, open a new issue with the Remote-SSH output channel
